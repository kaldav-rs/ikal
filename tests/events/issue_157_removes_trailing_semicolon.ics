BEGIN:VEVENT
DTSTAMP:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
DTSTART:20150325T101010
RRULE:FREQ=YEARLY;BYMONTH=11;BYDAY=1SU;
END:VEVENT
