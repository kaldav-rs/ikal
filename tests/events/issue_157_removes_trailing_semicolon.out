Ok(
    VEvent {
        dtstamp: Naive(
            2017-02-09T19:23:58,
        ),
        uid: Text {
            params: {},
            text: "5UILHLI7RI6K2IDRAQX7O",
        },
        dtstart: DateTime(
            Naive(
                2015-03-25T10:10:10,
            ),
        ),
        class: None,
        created: None,
        description: None,
        geo: None,
        last_modified: None,
        location: None,
        organizer: None,
        priority: None,
        sequence: None,
        status: None,
        summary: None,
        transp: None,
        url: None,
        recurid: None,
        rrule: Some(
            Recur {
                freq: Yearly,
                until: None,
                count: None,
                interval: None,
                by_second: [],
                by_minute: [],
                by_hour: [],
                by_day: [
                    WeekdayNum {
                        weekday: Sunday,
                        ord: Some(
                            1,
                        ),
                    },
                ],
                by_monthday: [],
                by_yearday: [],
                by_weekno: [],
                by_month: [
                    11,
                ],
                by_setpos: [],
                wkst: None,
            },
        ),
        dtend: None,
        duration: None,
        attach: [],
        attendee: [],
        categories: [],
        comment: [],
        contact: [],
        exdate: [],
        rstatus: [],
        related_to: [],
        resources: [],
        rdate: [],
        alarms: [],
        x_prop: {},
        iana_prop: {},
    },
)
